gluoncv/data/otb/tracking.py

```diff
@@ -0,0 +1,192 @@
+"""Visual Tracker Benchmark."""
+import json
+import os
+from glob import glob
+from tqdm import tqdm
+from mxnet.gluon.data import dataset
+from gluoncv.utils.filesystem import try_import_cv2
+cv2 = try_import_cv2()
+
+class Video(object):
+    """
+    Abstract video class. get video class information for example imgs.
+    Args:
```

Comment:
please follow the same doctoring convention in GluonCV, otherwise it may not render properly in website api